CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- New annotation shapes: polygons, polylines, points
+- Undo/redo feature
+- Grid to estimate size of objects
+- Context menu for shapes
+- A converter to PASCAL VOC format
+- A converter to MS COCO format
+- A converter to mask format
+- License header for most of all files
+- .gitattribute to avoid problems with bash scripts inside a container
+- CHANGELOG.md itself
```

Comment:
- Size of bounding boxes
- Color by label, by group features
- Ability to setup shapes opacity manually
- Split feature
- Grouping feature
- Propagate feature
- Full screen player

Reply:
```
.gitattribute to avoid problems with bash scripts inside a container.
```
 May be better move it to "Fixed"?